### ISSUE TITLE
Difficult positions

### ISSUE THREAD

**Issue by Matthies** [Matthies/RubiChess#304]
r1b2k1r/1pp1npp1/2qp3p/p7/P2b3P/1P1B1PP1/R4NK1/2BQR3 w - - 0 22 
Rubi misses Bc4 (master 28.8.2021, depth 51)

**Comment by tissatussa** [comment]
indeed, some other (SF-like) engines find Bc4 after a rather short time, giving eval about +1 (and other moves about 0.00) .. what can be learned from this test position (and others like it) ?  Can you tune parameters / piece-square-tables to make Rubi play better ?  Will it then lack strength in other positions ?  What is the purpose of this issue ?
